apps/web/src/components/integrations/add-integration-dialog.tsx

```
@@ -0,0 +1,156 @@
+            e.stopPropagation();
+            mcpForm.handleSubmit();
+          }}
+        >
+          <div className="grid gap-4 py-4">
+            <mcpForm.Field
+              name="name"
+              validators={{
+                onChange: ({ value }) =>
+                  !value.trim() ? 'Integration name is required' : undefined,
+              }}
+              children={(field) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)